beta/src/components/MDX/Sandpack/SandpackRoot.tsx

```diff
@@ -74,7 +74,7 @@ function SandpackRoot(props: SandpackProps) {
 
   files['/styles.css'] = {
     code: [sandboxStyle, files['/styles.css']?.code ?? ''].join('\n\n'),
-    hidden: !files['/styles.css']?.visible,
+    hidden: files['/styles.css']?.hidden,
```

Comment:
Couldn't find neither `css styles.css hidden` nor `css styles.css visible` so maybe this should be 
```suggestion
    hidden: true,
```
instead which would match the old behavior?